.devcontainer/Dockerfile

```diff
@@ -22,14 +22,12 @@ RUN if [ "${INSTALL_NODE}" = "true" ]; then su $USERNAME -c "umask 0002 && . /us
 
 # Install terraform
 ARG TERRAFORM_VERSION="1.0.5"
-RUN apt-get update && apt-get install -y gnupg software-properties-common curl \
-    && curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - \
-    && apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" \
-    && apt-get update && apt-get install -y terraform "${TERRAFORM_VERSION}"
+COPY .devcontainer/scripts/terraform.sh /tmp/
+RUN bash /tmp/terraform.sh "${TERRAFORM_VERSION}" /usr/bin
 
-ARG DOCKER_GROUP_ID
-COPY .devcontainer/scripts/docker-client.sh /tmp/
-RUN /tmp/docker-client.sh $USERNAME
+# ARG DOCKER_GROUP_ID
```

Comment:
Add these back in? (saves me PRing a small change)

Reply:
oops